cinema/models.py

```diff
@@ -1,10 +1,36 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=100)
+    last_name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
```

Comment:
The `max_length` for the `name` field in the `CinemaHall` model should be 100 to match the migration file. This ensures consistency and prevents potential data truncation issues.